main.go

```diff
@@ -20,10 +21,27 @@ type roundTripper struct {
 
 func (rt roundTripper) RoundTrip(r *http.Request) (*http.Response, error) {
 	r.Header.Set("Authorization", fmt.Sprintf("token %s", rt.accessToken))
-	transport := http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: rt.insecure}}
+	transport := rt.getTransport()
 	return transport.RoundTrip(r)
 }
 
+func (rt roundTripper) getTransport() *http.Transport {
+	proxyURLStr := os.Getenv("HTTPS_PROXY")
+	transport := &http.Transport{
```

Comment:
@David-Crty are your sure this should be pointer ?

```suggestion
	transport := http.Transport{
```